src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
> it comes back with our OG one regardless of entity type

This is the expected behavior - our selection handler is a wrapper around the original - entity specific - selection handler.